std/special/bootstrap.zig

```diff
@@ -43,6 +45,16 @@ nakedcc fn _start() noreturn {
     @noInlineCall(posixCallMainAndExit);
 }
 
+nakedcc fn wasiStart() noreturn {
+    // TODO: Decide if alloc at init is acceptable for args and env
+    // @llvm.wasm.mem.grow.i32
+    // __wasi_args_get()
+    // __wasi_args_sizes_get()
+    // __wasi_environ_get()
+    // __wasi_environ_sizes_get()
+    std.os.posix.exit(callMain());
```

Comment:
This can just use the wasi core API directly; no need to go through the POSIX API.